src/plugins/plugin.legend.js

```diff
@@ -237,26 +237,54 @@ var Legend = Element.extend({
 			minSize.height = me.maxHeight; // fill all the height
 		}
 
+		var getLineWidths = function(boxWidth, textLines) {
+			return textLines.map(function(textLine) {
+				return boxWidth + (fontSize / 2) + ctx.measureText(textLine).width;
+			});
+		};
+
 		// Increase sizes here
 		if (display) {
 			ctx.font = labelFont.string;
 
 			if (isHorizontal) {
+
 				// Labels
 
 				// Width of each line of legend boxes. Labels wrap onto multiple lines when there are too many to fit on one
 				var lineWidths = me.lineWidths = [0];
+				var maxHeight = 0;
 				var totalHeight = 0;
 
 				ctx.textAlign = 'left';
 				ctx.textBaseline = 'top';
 
 				helpers.each(me.legendItems, function(legendItem, i) {
 					var boxWidth = getBoxWidth(labelOpts, fontSize);
-					var width = boxWidth + (fontSize / 2) + ctx.measureText(legendItem.text).width;
+					var width;
+					var height;
+					var grossHeight;
+
+					if (helpers.isArray(legendItem.text)) {
+						var widths = getLineWidths(boxWidth, legendItem.text);
```

Comment:
I'd inline this variable since it's only used one place:

```
width = getLineWidths(boxWidth, legendItem.text).reduce(function(acc, v) {
	return v > acc ? v : acc;
}, 0);
```